src/nvim/testdir/test_lambda.vim

```diff
@@ -31,15 +31,13 @@ function! Test_lambda_with_timer()
   endfunction
 
   call s:Foo()
-  sleep 210ms
-  " do not collect lambda
-  call test_garbagecollect_now()
```

Comment:
why remove this? it is part of the upstream test.

Reply:
Happened when I played around and removed all potential error sources. Will add back.